spec/skill_spec.rb

```diff
@@ -1,60 +1,68 @@
 # frozen_string_literal: true
```

Comment:
Sorry. This file should be deleted.

Reply:
[Solved](https://github.com/KeroKnows/s-killer/pull/24/commits/c5252bf4e86894cd3e1c80b2c51a0324e7cfe7d0)